### PR TITLE
Fix anomaly releaser creating a "Stable "

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
Fixes this.
image
## Why It's Good For The Game
Bugfix
## Changelog
:cl:
fix: Anomaly releaser no longer creates a "Stable "
/:cl:

### CODE REVIEW

**Comment by Xander3359** [comment]
This item is awful and only spawns in the anomaly ruin, but that's not an excuse to have it not work.

The only thing it does is turn an anomaly core into a infinite anomaly.
I guess it could be useful for an antagonist to release an infinite pyro anomaly in medbay or something, but that isnt likely since it's from a ruin you have to stumble upon and be geared up enough to loot one properly.
Not to mention you need an actual core, not just the raw cores, and the space ruin doesnt have any way to refine cores.

File: code/game/objects/effects/anomalies/_anomalies.dm [hunk 1]
```diff
@@ -109,7 +109,7 @@
 ///Stabilize an anomaly, letting it stay around forever or untill destabilizes by a player. An anomaly without a core can't be signalled, but can be destabilized
 /obj/effect/anomaly/proc/stabilize(anchor = FALSE, has_core = TRUE)
 	immortal = TRUE
-	name = has_core ? "stable " : "hollow " + name
+	name = has_core ? "stable " + name : "hollow " + name
```
Comment (Time-Green) [inline code/game/objects/effects/anomalies/_anomalies.dm:112]:
```suggestion
	name = (has_core ? "stable " : "hollow ") + name
```

**Comment by Time-Green** [comment]
> This item is awful and only spawns in the anomaly ruin, but that's not an excuse to have it not work.
> 
> The only thing it does is turn an anomaly core into a infinite anomaly. I guess it could be useful for an antagonist to release an infinite pyro anomaly in medbay or something, but that isnt likely since it's from a ruin you have to stumble upon and be geared up enough to loot one properly. Not to mention you need an actual core, not just the raw cores, and the space ruin doesnt have any way to refine cores.

It's not supposed to be strong? It says what it does on the tin, and there's plenty of fun to be had with a stable anomaly

**Comment by Xander3359** [comment]
IMO this should be obtainable in some way outside of just 1 non-guaranteed space ruin.
Thats really the main complaint.

**Review by Time-Green — APPROVED** [review]

**Pull request merged**